src/pages/People.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect} from 'react';
+import {Person} from '../components/Person/Person';
+import {useDispatch, useSelector} from 'react-redux';
+import {dataLoad} from '../redux/dataLoad.js';
+
+//import people from "../people.json";
+import './page.css';
+
+export const People = () => {
+  const people = useSelector(state => state.data);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    function load() {
+      dispatch(dataLoad);
+    }
+    load();
```

Comment:
Why do you need to create a new function?
You can do `dispatch(dataLoad)` instead